### ISSUE TITLE
md-select is not sizing properly in IE11

### ISSUE THREAD

**Issue by rhnatiuk** [angular/material#3992]
![capture](https://cloud.githubusercontent.com/assets/5135587/9031355/9d940840-39ba-11e5-9352-43b18cf7a295.PNG)

This behavior is there for ages in IE11 (since 0.8.something, if I remember correctly) - wonder why no one is reporting it.

**Comment by Londovir** [comment]
It was reported, just hasn't ever been fixed officially. There's a workaround in issue #2769 where this was reported in the past (and I commented in that one as well). This issue is a duplicate of that one.

One thing I've noticed is if you are seeing odd IE11 sizing/layout issues, the first culprit typically is the `flex-basis` CSS style. IE11 has significant issues with that one, namely that if it is set to 0 (usually through things like `flex: 0`, it will not calculate layout correctly. I've seen this in #3926 as well.
